plugin/db/pg/pg.go

```diff
@@ -116,9 +116,7 @@ func guessDSN(username, password, hostname, port, database, sslCA, sslCert, sslK
 		"password": password,
 	}
 
-	if sslCA == "" {
-		m["sslmode"] = "disable"
```

Comment:
I would choose to comment out the code and add the comment.

According to https://www.postgresql.org/docs/current/libpq-ssl.html, the default "prefer" does not make sense and I guess that's the reason we set "disable" here. But looks like render and probably other service providers also conforms to the old behavior for the same compatibility reason.

Reply:
+1